QUICKSTART.md

```diff
@@ -0,0 +1,351 @@
+
+```bash
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-with-plugins.sh) --version v0.11.0 --folder $(pwd)/kcp --create-folder
+export PATH="$PATH:$(pwd)/kcp/bin"
+```
+
+Start **kcp** with the following command:
+
+```bash
+kcp start >& kcp_log.txt &
+export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+```
+
+After few seconds, check that `kcp` is running using the command:
+
+```bash
+kubectl version --short
+```
+
+which should yield something like:
+
+```console
+Client Version: v1.25.3
+Kustomize Version: v4.5.7
+Server Version: v1.24.3+kcp-v0.11.0
+```
+
+Additionally, one can check the available virtual workspaces using the command:
+
+```bash
+kubectl ws tree
+```
+
+which should yield something like:
+
+```console
+.
+└── root
+    └── compute
+```
+
+## 2. Install and run **KCP-Edge**
+
+Download the kcp **KCP-Edge** binaries and scripts into a `kcp-edge` subfolder in your current working directory using the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-edge.sh) --version v0.1.0 --folder $(pwd)/kcp-edge --create-folder
+export PATH="$PATH:$(pwd)/kcp-edge/bin"
+```
+
+Start **KCP-Edge** with the following command:
+
+```shell
+kcp-edge start --user kit
+```
+
+Check that `KCP-Edge` controllers are running with the following command:
+
+```shell
+ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+```
+
+which should yield something like:
+
+```console
+user     1892  0.0  0.3 747644 29628 pts/1    Sl   10:51   0:00 mailbox-controller --inventory-context=root --mbws-context=base -v=2
+user     1902  0.3  0.3 743652 27504 pts/1    Sl   10:51   0:02 scheduler -v 2 --root-user kcp-admin --root-cluster root --sysadm-context system:admin --sysadm-user shard-admin
+user     1912  0.3  0.5 760428 41660 pts/1    Sl   10:51   0:02 placement-translator --allclusters-context system:admin -v=2
+```
+
+Check that the Edge Service Provider Workspace (`espw`) is created with the following command:
+
+```shell
+kubectl ws tree
+```
+
+which should yield:
+
+```console
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## 3. Create a kcp-edge Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)
+
+**IMW** are used by **KCP-Edge** to store *sync targets* and *placement* objects.
+Create an **IMW** named `imw-1` with the following command:
+
+```shell
+kubectl ws root
+kubectl ws create "imw-1"
+```
+
+**WMW** are used by **KCP-Edge** to store *workloads* amd *edge placement* objects.
+Create an **WMW** named `wmw-1` in a `my-org` workspace with the following command:
+
+```shell
+kubectl ws root
+kubectl ws create "my-org"
+kcp-edge --create_wmw wmw-1  # replaces: ensure-wmw.sh "wmw-1"
```

Comment:
In my discussion with @francostellari  we came up with the following draft flag names:  

```
1. kcp-edge --create_wmw wmw-1
2. kcp-edge --syncer florin
3. kcp-edge --create_inv_item
```

Reply:
I think that mixes up commands and flags.  See https://github.com/spf13/cobra#concepts for an exposition of concepts.

I thought @francostellari preferred these to be kubectl plugins?

Reply:
"create" is not idempotent; "ensure" is idempotent.
Because the thing we want to be done here for a WMW is not just one step but several, it is more convenient for a user if there is an idempotent command rather than one that fails if the desired state has already been partly or fully achieved.